userapi/src/main/java/fk/prof/userapi/verticles/HttpVerticle.java

```diff
@@ -265,9 +269,15 @@ else if(result.cause() instanceof IllegalArgumentException) {
     }
 
     private AggregatedProfileNamingStrategy buildFileName(String appId, String clusterId, String procId,
-                                                          AggregatedProfileModel.WorkType workType, String startTime) {
+                                                          AggregatedProfileModel.WorkType workType, String startTime, String duration) {
         ZonedDateTime zonedStartTime = ZonedDateTime.parse(startTime, DateTimeFormatter.ISO_ZONED_DATE_TIME);
-        return new AggregatedProfileNamingStrategy(baseDir, 1, appId, clusterId, procId, zonedStartTime, aggregationWindowDurationInSecs, workType);
+        int windowDuration;
+        try {
```

Comment:
Don;t handle exception. Also lets use this opportunity to flush out all vestiges of aggregation window duration from userapi